monarchmoney/monarchmoney.py

```diff
@@ -2481,21 +2481,27 @@ async def gql_call(
             document=graphql_query, operation_name=operation, variable_values=variables
         )
 
-    def save_session(self, filename: str) -> None:
+    def save_session(self, filename: str = None) -> None:
```

Comment:
Set type to 'Optional[str]', when setting the default to None.

Reply:
Thanks for the review! I've fixed the typing hints